src/main/java/com/aws/greengrass/componentmanager/plugins/docker/DefaultDockerClient.java

```diff
@@ -166,6 +167,23 @@ private Optional<UserNotAuthorizedForDockerException> checkUserAuthorizationErro
         return Optional.ofNullable(error);
     }
 
+    /**
+     * Use docker command to delete the docker image
+     *
+     * @param image docker image to delete
+     * @throws DockerImageDeleteException if error is encountered
+     */
+    public void deleteImage(Image image) throws DockerImageDeleteException {
+        CliResponse response = runDockerCmd(String.format("docker rmi -f %s", image.getImageFullName()));
```

Comment:
Why do we need force deletion here?

Reply:
> Why do we need force deletion here?

Incase the container which use this image not stoped, so you mean this pattern would not happen?

Reply:
OK, will not use force deletion.